about.html

```diff
@@ -0,0 +1,150 @@
+              </li>
+              <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+              <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+              <li class="nav-item">
+                <a href="#sponsor" class="nav-link">Sponsor</a>
+              </li>
+              <li class="nav-item">
+                <a href="#" class="nav-link campain">CG Campain</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <main class="about-wrapper">
+      <section class="about">
+        <div class="about-us">
+          <span class="about-quote"
+            >“Our highest deeds come from helping the lowest people.”</span
+          >
+          <h1 class="about-goal">
+            “Overcoming poverty is not a gesture of charity. It is an act of
+            justice. It is the protection of a fundamental human right, the
+            right to dignity and a decent life.”
+          </h1>
+        </div>
+        <div class="about-ideal">
+          The Genuine Charity brings together the community of experts,
+          academics, nad activists who comprise the Creative Commins affiliates
+          network in a different country every two years. This year, we hope to
+          expand our invitation list including organizations and individuals who
+          want to work with us on shared projects taht advance the cause of yhe
+          commons, free culture nd open kwnoledge. So if you're active and
+          engaged in the worlds of open content and kwnoledge - free software
+          advocates. Wikipedias, Open Kwnoledge, galeries, libraries, museums,
+          archives, governments and foundations, kawyers, and activists - we
+          hope you'll consider joing us this year to build a stronger, more
+          vibrant commons together.
+        </div>
+        <div class="contact">
+          Please contact us per Email for any further questions about Genuine
+          Charity and care at
+          <strong><u>christianbiringanine22@gmail.com</u></strong> >
+        </div>
+      </section>
+      <section class="about-logo">
+        <div class="title">
+          <h1>Genuine Charity and Care Logo</h1>
+        </div>
+        <div class="concern">
+          <p>
+            The logo of Guine Charity and care will be decided through the logo
+            competiton form 8 June to 7 July. After online public voting and
+            Selection Committee's judging process. the logo that will win prize.
+            After consultation with Mr. Agrawal will be optimized the
+            prize-winning-logo for the coming years.
+          </p>
+        </div>
+      </section>
+      <section class="past-activities">
+        <h1 class="activ-title">See the past GC campain</h1>
```

Comment:
- [ ] **Required:**  [There should be one `h1` tag per page.](https://neilpatel.com/blog/h1-tag/#:~:text=H1%20tags%20are%20the%20main,be%20one%20H1%20per%20page.) The `About` and `Home` pages should contain more than one each.

Reply:
Okay :+1: 
correct that

Reply:
resolved 👍